traffic_ops/traffic_ops_golang/profileparameter/postparameterprofile.go

```diff
@@ -48,9 +49,28 @@ func PostParamProfile(w http.ResponseWriter, r *http.Request) {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New("posting parameter profile: "+err.Error()))
 		return
 	}
+	paramName, ok, err := getParamNameFromID(*paramProfile.ParamID, inf.Tx.Tx)
+	if err != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New("getting parameter name from id: "+err.Error()))
+	} else if !ok {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusNotFound, errors.New("parameter not found"), nil)
+	}
+	api.CreateChangeLogRawTx(api.ApiChange, "PARAM: "+paramName+", ID: "+strconv.FormatInt(*paramProfile.ParamID, 10)+", ACTION: Assigned "+string(len(*paramProfile.ProfileIDs))+" profiles to parameter", inf.User, inf.Tx.Tx)
```

Comment:
this doesn't look right

`PARAM: CONFIG proxy.config.proxy_name, ID: 14, ACTION: Assigned [ profiles to parameter`

Reply:
![image](https://user-images.githubusercontent.com/251272/62727944-d1c58200-b9d7-11e9-94ff-14377a5e1ace.png)